UPGRADING.rst

```diff
@@ -20,6 +20,16 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#5301 Alarm on detection of new vulnerabilities by Inspector
```

Comment:
This hunk is in the wrong commit.

Since this commit is a semantically neutral refactoring, I would not tag it `u`.